### DISCUSSION TITLE
Discution privée

### DISCUSSION THREAD

**Discussion by TmpName** [Kodi-vStream/venom-xbmc-addons#4064] [Support & aide]
Bon question bete, mais je ne trouve plus comment demarrer une discussion privé entre devs ?
Ca existe encore ?

**Comment by sizanic** [comment]
C'est au niveau de la team il me semble.

**Reply by sizanic** [reply]
Ah non. Il y a un message, les discussions de team ont été transféré en discussions.
![Screenshot_20230609-193129](https://github.com/Kodi-vStream/venom-xbmc-addons/assets/47307622/57c21895-d847-4b42-bb6e-267fee304950)

**Reply by TmpName** [reply]
Yep j'ai vu ca, apparement il y aurait une option pour rendre une discution privé (privée et publique se font de la meme maniere), mais je vois rien de special.

**Reply by sizanic** [reply]
En créant un nouveau repository privé.
Et des discussions dans ce repository.
Je n'ai pas les droits pour créer un repo privé au nom de l'organisation vStream.

**Reply by TmpName** [reply]
Bon ben j'ai pas trouvé, et j'ai pas pu deplacer cette discussion vers un autre repo.
J'ai aussi regardé au niveau des teams .....
Et j'ai trouvé comment creer un nouveau repo (en passant je t'ai rajouté des droits, ca devrait passer now) mais ca dit "You choose who can see and commit to this repository.", donc ca fait beacoup juste pour une discution.

Bon pas grave du coup ce sera public. Je ne suis pas sur discord, mais par hasard il n'y aurait pas des motives (ayant des connaissances en python) qui voudraient agrandir la team ?

Je leve le pied sur ce github, je ne suis pas le seul, et de voir la charge de travail sur certains (pour ne pas te citer directement) n'envisage rien de bon.

**Comment by sizanic** [comment]
Salut @TmpName ,

Si ton message est de nous informer que tu prends du recul avec ce projet, ton annonce peut être publique.
Bien au contraire, nos utilisateurs nous suivent et veulent savoir.

Tout comme @Arias800 l'avait annoncé il y a un an : https://github.com/Kodi-vStream/venom-xbmc-addons/issues/3514

Je vais m'occuper de faire la communication sur Discord pour chercher des motivés en Python 😄

**Reply by TmpName** [reply]
lol, non je prend pas du recul (enfin pas plus), ca c'est deja fait depuis un moment ^^, pas de changement de ce cote.

C'est juste que je me fais du soucis, et on trouvera pas sur le github, je me dis que la commu discord doit etre plus active ?

**Reply by sizanic** [reply]
ah ok, lol.

Coté Discord officiel vStream : https://discord.gg/KRsDT7PNHN
On y trouve plus d'interactions avec nos utilisateurs, en effet.
Et lorsqu'un bug y est effectivement découvert, j'envoi la personne ici pour créer une issue.
Quelques idées d'évolutions également sur le Discord.
<br><br>

Et également beaucoup d'interactions sur le Discord des alKODIques, qui est la communauté de référence des utilisateurs francophones de KODI : https://discord.gg/3RTRvT2PSE
(également disponible sur facebook et le site lesalkodiques.com)

Enormément de discussions autour de vStream, qui est reconnu comme la meilleure extension de streaming en français.

Donc en effet, le calme de ce github ne reflète pas l'engouement des utilisateurs de vStream.

Par contre, tu as bien raison de te faire du souci sur la pérennité si tout repose sur très très peu de personne.

<br><br><br>

_(cette communication contient un placement de produit)_
